packages/web/src/components/result/ReactiveList.js

```diff
@@ -411,7 +411,21 @@ class ReactiveList extends Component {
 
 	renderResultStats = () => {
 		if (this.props.onResultStats && this.props.total) {
-			return this.props.onResultStats(this.props.total, this.props.time);
+			const results = parseHits(this.props.hits) || [];
+			const streamResults = parseHits(this.props.streamHits) || [];
+			let filteredResults = results;
+
+			if (streamResults.length) {
+				const ids = streamResults.map(item => item._id);
+				filteredResults = filteredResults.filter(item => !ids.includes(item._id));
+			}
+
+			return this.props.onResultStats(
+				this.props.total,
```

Comment:
It would be best if we return an object instead:

```js
const stats = {
    totalPages: <>,
    currentPage: <>,
    ...
}

this.props.onResultStats(stats);
```

This will improved the DX significantly.

Reply:
True, lets do that 👍